hera_pspec/uvpspec_utils.py

```diff
@@ -945,3 +946,145 @@ def _fast_lookup_blpairts(src_blpts, query_blpts, time_prec=8):
     blpts_idxs = np.where(src_blpts == query_blpts[:,np.newaxis])[1]
 
     return blpts_idxs
+
+
+def _get_red_bls(uvp, bl_len_tol=1., bl_ang_tol=1.):
```

Comment:
seems like no need for these to be "hidden" functions? These seem well-documented and useful, so might as well make the un-underscored?

Reply:
They're exposed as methods in the `UVPSpec` object, so they should be reasonably visible. I just put them here to offload some code into `uvpspec_utils` (like some similar functions in this file).

Reply:
ok great